package.json

```diff
@@ -96,8 +97,7 @@
   },
   "nyc": {
     "include": [
-      "bin/**.js",
-      "packages/**/*.js"
+      "bin/**.js"
```

Comment:
Don't touch this, it should be done in PR where we store coverage after removing all tests between v3 and v4